src/librustdoc/html/render/mod.rs

```diff
@@ -3548,9 +3550,6 @@ fn render_assoc_items(
             );
         }
     }
-    if let AssocItemRender::DerefFor { .. } = what {
-        return;
-    }
```

Comment:
This now documents traits implemented by the Deref target?

Reply:
Hmm, I am not quite sure what you're asking here... The goal of this particular portion is to allow running `render_deref_methods` even when we're already inside one level of `Deref` targetting. Are you concerned that this might cause more than just the method from the `Deref` target to appear?

Reply:
I'm mostly just trying to figure out what this is doing :laughing: Ignore this comment, I think https://github.com/rust-lang/rust/pull/80653/files/9accf1d281d676b81753892b193bcd20a49700c1#r551024620 asks it better.

Reply:
Okay, I know it's a bit confusing, or at least it was when I was trying to work through it... 😅 Of course I'm happy to add any comments or whatever else might help.